fenix/app/src/main/java/org/mozilla/fenix/debugsettings/tabs/TabTools.kt

```diff
@@ -208,7 +328,31 @@ private fun TabToolsPreview(
                     privateTabCount = privateTabCount,
                     totalTabCount = totalTabCount,
                     inactiveTabsEnabled = inactiveTabsEnabled,
+                    onAddTabsToActiveClick = {
+                        scope.launch {
+                            snackbarState.showSnackbar("Add $it tabs to active")
+                        }
+                    },
```

Comment:
I'm debating adding logic to this preview to have the buttons also update the tab counter UI. (this is one of the downsides of removing `BrowserStore` as a parameter, as this update may have been handled for us.

Reply:
I'm cool with this. We'll want to make `TabTools` depend on a boolean again then for `inactiveTabsEnabled`, instead of passing our `Settings` mega singleton as a dependency.